src/app/_components/bottom-navibar.tsx

```
@@ -0,0 +1,48 @@
+  return (
+    <nav className="bg-gray-black mt-0.875 mb-0.875 fixed bottom-0 left-1/2 z-50 flex h-17.5 w-full max-w-[500px] -translate-x-1/2 transform items-center justify-around pt-2.25 pb-4">
+      {navigationTabs.map((tab) => {
+        const active = isActive(tab.path);
+        const IconComponent = tab.Icon;
+
+        return (
+          <Link
+            href={tab.path}
+            className="relative flex w-20 flex-col items-center gap-2"
+            aria-label={tab.label}
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)